ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import Box from '../../../ui/box';
+import Typography from '../../../ui/typography';
+import CheckBox from '../../../ui/check-box';
+import I18nValue from '../../../ui/i18n-value';
+import {
+  COLORS,
+  DISPLAY,
+  FLEX_DIRECTION,
+  TYPOGRAPHY,
+} from '../../../../helpers/constants/design-system';
+import { getAdvancedGasFeeValues } from '../../../../selectors';
+import { setAdvancedGasFee } from '../../../../store/actions';
+
+import { useAdvancedGasFeePopoverContext } from '../context';
+
+const AdvancedGasFeeDefaults = () => {
+  const dispatch = useDispatch();
+
+  const {
```

Comment:
What is the difference between the advanced settings which are kept in the context and the advanced settings which are kept in Redux? Is the difference that Redux represents the saved version and the context represents the changed-but-unsaved version? If that's the case, then I'm wondering whether it makes more sense to include that in the name somewhere? Maybe we have `unsavedBaseFeeMultiplier` and `unsavedMaxPriorityFeePerGas` vs. `savedBaseFeeMultiplier` and `savedMaxPriorityFeePerGas`. I feel like these names could get confusing.

Reply:
I was just following the existing pattern in the`AdvancedGasFeePopoverContextProvider` changing them might end up with unrelated changes to this Pr, if necessary we can have a follow-up PR!!